### ISSUE TITLE
Unexpected rounding behaviour.

### ISSUE THREAD

**Issue by atrabattoni** [stencillogic/astro-float#23]
I'm surprised by the rounding behaviour used in BigFloat.round(...). rounding 0.99 -> 0 seems very unnatural to me. 

```rust 
use astro_float::{BigFloat, RoundingMode};

fn main() {
    let result = BigFloat::from_f64(0.99, 64).round(0, RoundingMode::ToEven);
    println!("{result}");
}
```
Output:

```
0.0
```

**Comment by atrabattoni** [comment]
It looks like something is broken between -1.0 and 1.0. Other values seems to work fine.

**Comment by stencillogic** [comment]
Hi, yes, looks like a problem somewhere in rounding function.

**Comment by stencillogic** [comment]
Thank you for raising the issue.

**Comment by stencillogic** [comment]
The fix has been applied, the changes are published as v0.9.4. Is there anything left as part of this issue?

**Comment by atrabattoni** [comment]
I tried the latest version, the bug seems to be fixed thanks. I think this issue can be considered closed.
